edb/compose-inherit/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+mod py_children;
+mod py_inherit;
+
+use proc_macro::TokenStream;
+use quote::format_ident;
+use syn::parse_macro_input;
+
+/// Takes a struct declaration and emits:
+/// - struct declaration (basically) unchanged,
```

Comment:
what means "basically unchanged"? If it's unchanged, you don't need macro attribute, derive macro is okay (even emitting extra structures is fine)

Reply:
It removes `#[pychildren]` attribute.

Reply:
Oh, I could just define the `#[pychildren]` macro as a no-op. And then it could really be unchanged. And then this could be a derive macro - if I didn't need the attributes.